nucypher/policy/conditions/lingo.py

```diff
@@ -83,12 +83,13 @@ class InvalidExpression(ValueError):
     class ReturnValueTestSchema(CamelCaseSchema):
         comparator = fields.Str()
         value = fields.Raw(allow_none=False)  # any valid type (excludes None)
+        key = fields.Raw(allow_none=True)
 
         @post_load
         def make(self, data, **kwargs):
             return ReturnValueTest(**data)
 
-    def __init__(self, comparator: str, value):
+    def __init__(self, comparator: str, value, key: Union[int, str] = None):
```

Comment:
`Optional` type hint here?

Reply:
i got linting errors....hold on...

Reply:
error occurred between the chair and the keyboard